src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1834,7 +1834,6 @@ internal enum ErrorCode
         ERR_StaticAnonymousFunctionCannotCaptureThis = 8821,
         ERR_OverrideDefaultConstraintNotSatisfied = 8822,
         ERR_DefaultConstraintOverrideOnly = 8823,
-        WRN_PartialMethodTypeDifference = 8824,
```

Comment:
I think you mark these with `// Available comments`?

Reply:
Yes, we could. But this PR simply reverts a previous change so I'll leave as is.